source/hxvlc/util/Handle.hx

```diff
@@ -1,7 +1,7 @@
 package hxvlc.util;
 
 #if (!cpp && !(desktop || mobile))
-#error 'The current target platform isn\'t supported by hxvlc.'
+// #error 'The current target platform isn\'t supported by hxvlc.'
```

Comment:
Why?

Reply:
Was an accident from testing - would not recognize the platform on my local branch, so the linter wouldnt work.